draft-ietf-quic-recovery.md

```diff
@@ -846,6 +843,29 @@ DetectLostPackets(largest_acked):
     else if (loss_time == 0 && delay_until_lost != infinite):
       loss_time = now() + delay_until_lost - time_since_sent
```

Comment:
i think this might be cleaner to split this up to the end of the loop.

it looks like the intention is to the early retransmit timer to be the loss time of the first packet not lost by packet ordering. It might be cleaner like

```
 last_unacked = largest_acked.packet_number
 foreach (unacked < largest_acked.packet_number):
     ....
     last_unacked = unacked
 if (last_unacked < largest_acked.packet_number) :
    lost_time = now() + delay_until_lost - sent_packets[last_unacked].time_since_sent
```

and similarly in the other one as well

Reply:
sorry i meant first_unacked.